src/components/MultiHopTrade/components/AssetSelection.tsx

```diff
@@ -31,14 +30,9 @@ const TradeAssetAwaitingAsset = () => {
 
 type TradeAssetSelectProps = {
   assetId?: AssetId
-  onAccountIdChange: AccountDropdownProps['onChange']
-  accountId?: AccountId | undefined
-  accountSelectionDisabled?: boolean
+  isReadOnly?: boolean
```

Comment:
This is used in lending as well and will break things there- should we make it so the signature is left untouched for current implementors?

Reply:
Hmm i checked that those props were all unused, is there a plan to implement them later? Happy to add them back in, but only if there is a plan to actually use them rather than pass undefined everywhere

Reply:
That's correct, nice one -  they are currently expected in, but are not actually consumed at runtime which is were the confusion came from. Happy to remove these since there's no reason to have them currently and lending indeed passes stubs